src/server.c

```diff
@@ -150,62 +116,16 @@ void
 sig_handler(int sig) {
     if (force_exit)
         exit(EXIT_FAILURE);
-
-    char sig_name[20];
-    get_sig_name(sig, sig_name);
-    lwsl_notice("received signal: %s (%d), exiting...\n", sig_name, sig);
+    lwsl_notice("received signal: %s (%d), exiting...\n", strsignal(sig), sig);
```

Comment:
I do not understand why this is in the code at all.

You can ktrace/strace etc to check incoming signal on your process and do not need this at all unless for debugging. What is this for ? Why kind of log parsing will you do behind and how the OS dependance got any effect ?